models/Asset/Service.php

```diff
@@ -597,7 +597,7 @@ public static function getImageThumbnailByArrayConfig(array $config): null|Image
 
                 //check if high res image is called
 
-                preg_match("@([^\@]+)(\@[0-9.]+x)?\.([a-zA-Z]{2,5})@", $config['filename'], $matches);
+                preg_match("@([^\@]+)(\@[0-9.]+x)?\.(.*)\.([a-zA-Z]{2,5})@", $config['filename'], $matches);
```

Comment:
```suggestion
                preg_match("@([^\@]+)(\@[0-9.]+x)?\.([^\.]+)\.([a-zA-Z]{2,5})@", $config['filename'], $matches);
```
I'd prefer to make the pattern less generic.

Reply:
@melissakittl can you please confirm that my change works for you as well? Thanks!

Reply:
@brusch Works for me! Thanks :)

Reply:
Escaping the `.` is not even necessary. `[^.]+` should be enough.